<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1350</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1350</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Montserrat</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(3, 5, 21);
border-color: rgb(85, 255, 0);
color: rgb(160, 160, 160);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(30, 215, 96);
border-color: rgb(30, 215, 96);
selection-color: rgb(30, 215, 96);</string>
    </property>
    <property name="value">
     <number>99</number>
    </property>
    <property name="sliderPosition">
     <number>99</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(30, 215, 96);
font: 28pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(30, 215, 96);</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(30, 215, 96);</string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(30, 215, 96);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ARTISTAS">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
     </font>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ALBUMS">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="CANCIONES">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
     </font>
    </property>
    <property name="text">
     <string>Artista</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
     </font>
    </property>
    <property name="text">
     <string>Album</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
     </font>
    </property>
    <property name="text">
     <string>Cancion</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_currentSong">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="img_playlist">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>661</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_estructura">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>631</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Buscar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Eliminar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="playlist_type">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="accionMontar"/>
   </widget>
   <widget class="QMenu" name="menuTop_s">
    <property name="title">
     <string>Top's</string>
    </property>
    <addaction name="actionTop_5_Artistas"/>
    <addaction name="actionTop_5_Canciones"/>
    <addaction name="actionTop_5_Canciones_2"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Acerca de</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuTop_s"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string>background-color: rgb(3, 5, 21);
border-color: rgb(85, 255, 0);
color: rgb(160, 160, 160);
</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="accionMontar">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
  </action>
  <action name="actionTop_5_Artistas">
   <property name="text">
    <string>Top 5 Artistas</string>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
  </action>
  <action name="actionTop_5_Canciones">
   <property name="text">
    <string>Top 5 Albums</string>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
  </action>
  <action name="actionTop_5_Canciones_2">
   <property name="text">
    <string>Top 5 Canciones</string>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
